scripts/pkg-tools/which-pkg-manager

```diff
@@ -14,19 +14,19 @@ PACMAN=/usr/bin/pacman
 
 check_primary_rpm()
 {
-    test -x $RPM && $RPM -qf $RPM | sed -e 's/-.*//'
+    test -x "$RPM" && $RPM -qf "$RPM" | sed -e 's/-.*//'
 }
 
 
 check_primary_dpkg()
 {
-    test -x $DPKG && $DPKG -S $DPKG | grep '^dpkg:' | sed -e 's/:.*//'
+    test -x "$DPKG" && $DPKG -S "$DPKG" | grep '^dpkg:' | sed -e 's/:.*//'
 }
 
 
 check_primary_pacman()
 {
-    test -x $PACMAN && $PACMAN -Qo $PACMAN | grep "is owned by pacman" | sed -e 's/^.*is owned by //' -e 's/ .*//'
+    test -x "$PACMAN" && $PACMAN -Qo "$PACMAN" | grep "is owned by pacman" | sed -e 's/^.*is owned by //' -e 's/ .*//'
```

Comment:
This is a prime example why this "just wrap all uses of the shell variable in quotes" that _shellcheck_ suggests falls apart: Imagine `$PACMAN` would really contain weird characters that are being split into separate words. What would happen if that command is actually executed? There is no quote in the actual command invocation. If `$PACMAN` really contains a blank (`/usr/bin/pac man`), then what? Does the command invocation here include another level of shell expansion, so it would treat the first part as the command (`/usr/bin/pac`) and the second (`man`) as the first argument? Or would it really invoke the command `/usr/bin/pac man`?

No matter what, if the system is really that broken, it will fail on many levels; no worries about this script not returning the expected result.